crashtest.c

```diff
@@ -10,43 +10,46 @@
 #include "php_crashtest.h"
 #include "Zend/zend_alloc.h"
 
-uint32_t lineno;
+volatile uint32_t lineno = 0;
 
+static bool has_opline(zend_execute_data *execute_data)
+{
+    return execute_data
+        && execute_data->func
+        && ZEND_USER_CODE(execute_data->func->type)
+#if CFG_STRICT_OPLINE
+#else
```

Comment:
This is so you can either not define it (default) or define it to a falsey expression and you get the same result either way.